premake5.lua

```diff
@@ -11,7 +11,7 @@ workspace "BigBaseV2"
 	outputdir = "%{cfg.buildcfg}"
 
 	CppVersion = "C++17"
-	MsvcToolset = "v143"
+	MsvcToolset = "v142"
```

Comment:
Revert this to v143.

Reply:
Legit had VSCode open, and forgot to save revertion.